templates/common/_base/files/configure-ovs-network.yaml

```diff
@@ -490,6 +490,55 @@ contents:
       echo ""
     }
 
+    get_ip_from_ip_hint_file() {
+      local ip_hint_file="$1"
+      if [[ ! -f "${ip_hint_file}" ]]; then
+        return
+      fi
+      ip_hint=$(cat "${ip_hint_file}")
+      echo "${ip_hint}"
+    }
+
+    # This function waits for ip address set in node-ip service and used as hint
+    # to define br-ex to be bindable
+    # This is workaround for OCPBUGS-673 as it will not allow starting crio
+    # before address is bindable
+    try_to_bind_address() {
+      local ip_hint_file="$1"
+      ip_hint=$(get_ip_from_ip_hint_file "${ip_hint_file}")
+      if [[ -z "${ip_hint}" ]]; then
+        return
+      fi
+      while true; do
+        echo "Trying to bind ${ip_hint}"
+        exit_code=$(timeout 1s nc -l ${ip_hint}; echo $?)
+        if [[ exit_code -eq 124 ]]; then
+          echo "Address bound successfully"
+          break
+        fi
+        sleep 1
+      done
+    }
+
+    # Get interface that matches ip from node ip hint file
+    # in case file not exists return nothing
+    get_nodeip_interface() {
```

Comment:
So I would probably rename this to `get_nodeip_hint_interface`
and `get_default_interface` to `get_nodeip_interface`
and doc that `get_nodeip_interface` fallbacks to the default interface if no `nodeip_hint_interface` was found.